node/test/service/src/lib.rs

```diff
@@ -160,7 +174,7 @@ pub fn node_config(
 		keep_blocks: KeepBlocks::All,
 		transaction_storage: TransactionStorageMode::BlockBody,
 		chain_spec: Box::new(spec),
-		wasm_method: WasmExecutionMethod::Interpreted,
+		wasm_method: WasmExecutionMethod::Compiled,
```

Comment:
Is there any reason we should stick to `interpreted` ? CC @bkchr

Reply:
No.